deployments/kubernetes/chart/reloader/templates/clusterrole.yaml

```diff
@@ -26,7 +26,7 @@ rules:
       - list
       - get
       - watch
-{{- if .Capabilities.APIVersions.Has "apps.openshift.io/v1" }}
+{{- if or (.Capabilities.APIVersions.Has "apps.openshift.io/v1") .Values.isOpenshift }}
```

Comment:
it would be good if round bracket is added around `.Values.isOpenshift` param, just to make the or condition more understandable.

Reply:
Okay. But just for clarification:
`.Capabilities.APIVersions.Has` is a function in itself which needs 1 parameter. I had to explicitly add a bracket around it so that it doesn't complain of having 2 variables being passed into it. Otherwise we don't normally add brackets to or statement's variables unless they are compound or function calls

Reply:
Updated